src/cascadia/TerminalConnection/AzureConnection-ARM64.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include "AzureConnection.g.h"
+
+namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
+{
+    struct AzureConnection : AzureConnectionT<AzureConnection>
+    {
+        AzureConnection() = default;
+
+        AzureConnection(uint32_t rows, uint32_t columns);
+        static bool IsAzureConnectionAvailable();
```

Comment:
I'm wondering if it makes sense to get rid of the cpp for the arm version of this and have all the implementations just in the header, since they're all just E_NOTIMPL

Reply:
They're all not implemented _except_ IsAzureConnectionAvailable, which always returns false in arm64

Reply:
er, yea that's more technically correct. Point still stands though :)

Reply:
Gonna resolve this since we tried and ended up in a cppwinrt mess